tests/distributed/test_comm_ops.py

```diff
@@ -16,6 +16,10 @@
 @ray.remote(num_gpus=1, max_calls=1)
 def all_reduce_test_worker(tensor_parallel_size: int, rank: int,
                            distributed_init_port: str):
+    import os
+    del os.environ["CUDA_VISIBLE_DEVICES"]
```

Comment:
a small comment saying ray sets `CUDA_VISIBLE_DEVICES`.

i think this test passed originally because cupy doesn't respect that?

Reply:
fixed in [cefae38](https://github.com/vllm-project/vllm/pull/3625/commits/cefae388f90a44a18a8c973e285562dbab1b6374).